docs/js/custom.js

```diff
@@ -85,6 +85,18 @@ $(document).ready(function() {
         apiKey: '21ce3e522455e18e7ee16cf7d66edb4b',
         indexName: 'ezplatform',
         inputSelector: '#search_input',
+        transformData: function(hits) {
+            $.each(hits, function(index, hit) {
+                for (lvl=2; lvl<=3; lvl++) {
+                    if (null !== hit.hierarchy['lvl'+lvl]) {
+                        hits[index].hierarchy['lvl' + lvl] = hit.hierarchy['lvl' + lvl].replace(/[.¶]/g, '');
+                    }
+                    if ('undefined' !== typeof hit._highlightResult.hierarchy['lvl'+lvl]) {
+                        hits[index]._highlightResult.hierarchy['lvl'+lvl].value = hit._highlightResult.hierarchy['lvl'+lvl].value.replace(/[.¶]/g, '');
```

Comment:
`.replace(/[.¶]/g, '')` was copy-pasted from [search.js](https://github.com/ezsystems/developer-documentation/blob/7e2ede022a4fa2163b0968e130c41b19f551df0d/docs/api/rest_api_reference/assets/js/search.js#L158). Not sure about removing `.` dots characters.

```suggestion
                        hits[index]._highlightResult.hierarchy['lvl'+lvl].value = hit._highlightResult.hierarchy['lvl'+lvl].value.replace('¶', '');
```

Reply:
Remove only ¶ in e53d616290163bce16706092544db50b4ac7648f